app-localize-behavior.html

```diff
@@ -227,6 +232,12 @@
     };
   },
 
+  __languageChanged: function(newLang, oldLang) {
+    if (newLang === oldLang) return;
+    this.language = newLang;
+    this.fire('app-localize-behavior-language-changed', newLang);
```

Comment:
How is this different than just adding `notify: true` to the `language` property? Then, that will fire a `language-changed` event: https://www.polymer-project.org/1.0/docs/devguide/properties#notify